bzkanban.js

```diff
@@ -1062,32 +1079,48 @@ function dropCard(ev) {
     }
 }
 
-function dropBacklog(ev) {
-    if (ev.target.classList.contains("drop-target")) {
-        ev.target.classList.remove("drop-target-hover");
-    }
-    ev.preventDefault();
+function showBacklog() {
+    var button = document.getElementById("btnShowBacklog");
+    var backlogCol = document.querySelector("#BACKLOG.board-column");
+
+    if (backlogCol.style.display === "none" || null) {
+        // Load backlog on first access.
+        var backlog = backlogCol.querySelector(".cards");
+        if (backlog.children.length == 0) {
+            loadBacklogCards();
+        }
 
-    var bugCurrent = JSON.parse(ev.dataTransfer.getData("text"));
+        backlogCol.style.display = null;
+        button.innerText = "Hide Backlog";
+    }
+}
 
-    var bugUpdate = {};
-    bugUpdate.id = bugCurrent.id;
-    bugUpdate.status = "CONFIRMED";
-    bugUpdate.priority = bzDefaultPriority;
+function hideBacklog() {
+    var button = document.getElementById("btnShowBacklog");
+    var backlogCol = document.querySelector("#BACKLOG.board-column");
 
-    if (bzAddCommentOnChange) {
-        showBugModal(bugCurrent, bugUpdate);
-    } else {
-        writeBug(bugUpdate);
+    if (backlogCol.style.display === "" || null) {
+        var backlog = backlogCol.querySelector(".cards");
+        backlogCol.style.display = "none";
+        button.innerText = "Show Backlog";
     }
 }
 
-function showBacklog() {
-    document.getElementById("textBacklog").style.display = "inline-block";
-}
+function loadBacklogCards() {
+    showSpinner();
+    httpGet("/rest.cgi/bug?product=" + bzProduct + bzBacklogSearch, function(response) {
```

Comment:
Hmm actually...it'd be really nice if we could re-use the `bzRestGetBugsUrl ` value here and just tack on `resolution=---`. The only problem with that is the `target_milestone` wouldn't be right. So close.